packages/tasit-action/src/ethers.test.js

```diff
@@ -22,8 +22,12 @@ describe("ethers.js", () => {
     expect(wallet.address).to.have.lengthOf(42);
     expect(wallet.provider).to.be.not.undefined;
 
-    contract = new ethers.Contract(contractAddress, contractABI, wallet);
-    expect(contract.address).to.be.equals(contractAddress);
+    sampleContract = new ethers.Contract(
+      sampleContractAddress,
+      contractABI,
+      wallet
+    );
+    expect(sampleContract.address).to.be.equals(sampleContractAddress);
```

Comment:
chai expectations should mirror English grammar. "to be equals" doesn't work. Either "to equal" or "to be equal to" would

Reply:
```suggestion
    expect(sampleContract.address).to.equal(sampleContractAddress);
```